### PR TITLE
complte

### PR DESCRIPTION
**Pull request by Chi-Hime**

### CODE REVIEW

File: addnewcoin.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+				<div class="col-sm-10">
+				  <input type="text" class="form-control" id="inputYear" name="inputYear" placeholder="Coin Year" 
+					required minlength="4" maxlength="4">
+				</div>
+			  </div>
+			  
+			  <div class="form-group">
+				<label for="inputImage" class="col-sm-2 control-label">Image</label>
+				<div class="col-sm-10">
+				  <input type="file" name="inputImage" accept="image/*">
```
Comment (pixelfish22) [inline addnewcoin.php]: this was very ambitious and you're off to a great start. Are you following a tutorial or would you like some suggestions?

File: addnewcoin.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+				</div>
+			  </div>
+			  
+			  
+			  <div class="form-group">
+				<div class="col-sm-offset-2 col-sm-10">
+				  <button type="submit" class="btn btn-primary">&nbsp;&nbsp;&nbsp;Add&nbsp;&nbsp;&nbsp;</button>
```
Comment (pixelfish22) [inline addnewcoin.php]: better to use css padding than empty spaces

File: addnewcoin.php [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+	
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+	<!-- Latest compiled and minified JavaScript -->
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+	
+	
+	<script>
+		$("#form_addnewcoin").validate();
```
Comment (pixelfish22) [inline addnewcoin.php]: this should be fine but you should generally be in the habit of putting all your jquery in a $document.ready function

File: editcoin.php [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+						
+						// Create connection
+						$conn = new mysqli($servername, $username, $password, $dbname);
+						// Check connection
+						if ($conn->connect_error) {
+							die("Connection failed: " . $conn->connect_error);
+						}
+						 
+						
+						$sql = "select c.coinid, c.coinyear, s.state_name, s.state_abbr from state_table as s join coin_table as c on s.state_abbr = c.state_abbr where c.coinid = " . $coinid;
```
Comment (pixelfish22) [inline editcoin.php]: pretty fancy sql! :+1:

File: index.php [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+	<!-- Latest compiled and minified JavaScript -->
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+  </body>
+</html>
+
+
+<?php
+
+	$conn->close();
```
Comment (pixelfish22) [inline index.php]: nice job! easy to forget to close the connection

File: phpinfo.php [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<?php
```
Comment (pixelfish22) [inline phpinfo.php]: you should remove this -- it gives hackers too much useful information

File: test.php [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+$servername = "teddybridgewatervikings.com";
```
Comment (pixelfish22) [inline test.php]: this definitely should be removed from your repo

**Comment by pixelfish22** [comment]
Looks really good! You will most likely want to remove this repo now that it is graded b/c it has password data in it. But what I saw looked really good.
